check_pip_requirements.sh

```diff
@@ -19,7 +19,8 @@ do
         cat $new_pinned
         echo '--------------------------------------'
         echo "$pinned is no longer up to date with $reqs"
-        echo "Please regenerate the pinned requirements file."
+        echo "Please regenerate the pinned requirements file by running:"
+        echo "> make $pinned"
```

Comment:
Interestingly, when I `make hail/python/dev/pinned-requirements.txt`, I get the following: 
```bash
$ make hail/python/dev/pinned-requirements.txt
./generate-linux-pip-lockfile.sh hail/python/dev
+ PIP_COMPILE_IMAGE=hail-pip-compile:latest
++ docker images -q hail-pip-compile:latest
+ [[ 3821cc1c9757 == '' ]]
+ package=hail/python/dev
+ docker run --rm -it -v :/hail hail-pip-compile:latest pip-compile --upgrade hail/hail/python/dev/requirements.txt --output-file=hail/hail/python/dev/pinned-requirements.txt
Usage: pip-compile [OPTIONS] [SRC_FILES]...
Try 'pip-compile -h' for help.

Error: Invalid value for '[SRC_FILES]...': Path 'hail/hail/python/dev/requirements.txt' does not exist.
make: *** [Makefile:82: hail/python/dev/pinned-requirements.txt] Error 2
```

Reply:
Ahh, need to define `HAIL`